### PR TITLE
Js url from containers

### LINKED ISSUE #555: Allow containers to access Dusty Specified Urls
Since Dusty is now running nginx in a container, it should now be pretty easy to allow all containers to access these urls.  The best way to do this is probably to add entries into all containers `/etc/hosts` file that redirects that url out of the container to the vm.  This would then hit the nginx exposed port and go through nginxs routing layer to the correct container.

### PR DESCRIPTION
**Pull request by jsingle**
@paetling 

Resolves #555

### CODE REVIEW

File: dusty/command_file.py [hunk 1]
```diff
@@ -39,8 +39,17 @@ def _tee_output_commands(command_to_tee):
     ]
     return commands
 
-def _get_once_commands(app_spec):
-    once_commands = app_spec['commands']['once']
+def _hosts_export_commands(port_spec):
+    if not port_spec['hosts_file']:
+        return []
+    commands = ['DOCKERHOST=`/sbin/ip route|awk \'/default/ { print $3 }\'`']
```
Comment (paetling) [inline dusty/command_file.py]: isn't this just a fixed ip defined by docker?

Comment (jsingle) [inline dusty/command_file.py]: I don't think so.  The interwebs as well as our codebase told me to do this

**Comment by paetling** [comment]
ok. seems legit.  this is pretty awesome

**Comment by jsingle** [comment]
Yeah it was real easy to implement, you were right

File: docs/changelog.md [hunk 2]
```diff
@@ -2,6 +2,9 @@
 
 ## 0.6.2 (In Progress)
 
+* **New**
+    * Host names specified in app specs (`host_forwarding.host_name`) can now be used from inside any Dusty app container - previously they only worked from the host Mac.
```
Comment (thieman) [inline docs/changelog.md]: Grammar

**Pull request merged**